frontend/src/components/App.js

```diff
@@ -16,63 +19,66 @@ class App extends React.Component {
         }
     }
 
+    abortController = new window.AbortController();
+
     componentDidMount() {
-        fetch("http://localhost:5000/api/calendar/", {
-            method: 'POST',
-            body: JSON.stringify(this.state.start)
-        }) //todo move to prop file
+        let url = new URL(ADDRESS + API);
+        let params = [['start', this.state.start.toISOString()]];
+        url.search = new URLSearchParams(params).toString();
+
+        //todo move to prop file
+        fetch(url)
             .then(res => res.json())
             .then(json => {
-                this.setState({events: json})
+                this.setState({events: json});
             });
     }
 
-    async componentDidUpdate(prevProps, prevState) {
-        if (this.state.start !== prevState.start) {
```

Comment:
We need this comparison. If we don't compare dates. App will send request every time it renders

Reply:
Fixed. Sorry, did it by misunderstanding of how abort controller works.